themes/ddbasic/sass/components/user.scss

```diff
@@ -36,6 +36,7 @@
     }
     h1 {
       @include font('display');
+      float: left;
```

Comment:
Is it intentional that this float is added?

It does not seem to be related to the issue at hand and it is not mentioned in the commit message either.

Reply:
No - this was a mistake and has been removed in new commit

Reply:
:+1: